resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -1,21 +1,59 @@
 #!/usr/bin/env bash
+pid=$(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
+
 set -e
```

Comment:
I think you've added enough error handling to start, that we should probably just lose all of the `set -e` `set +e` calls. I don't think there are any spots left in this script that could return non-zero really.